### PR TITLE
AAP-4928: Show how to download public key in AH UI

### PR DESCRIPTION
**Pull request by lmalivert**
Jira: [AAP-4928](https://issues.redhat.com/browse/AAP-4928)

- New section on how to download public key from automation hub UI.
- Draft in [google doc](https://docs.google.com/document/d/1LwNRNryZlf_3ygAXG3jQQYxE9qkwLhisFOXifSIZ6x0/edit#)

### CODE REVIEW

**Review by rochacbruno — APPROVED** [review]

File: downstream/modules/automation-hub/proc-downloading-signature-public-keys.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+//this module appears in assembly-collections-and-content-signing-in-pah
+
+[id="proc-downloading-signature-public-keys"]
+
+= Downloading signature public keys
+
+After you sign and approve collections, download the signature public keys from the {HubName} UI. You must download the public key before you add it to the local system keyring.
```
Comment (jself-sudoku) [inline downstream/modules/automation-hub/proc-downloading-signature-public-keys.adoc:7]: Hope this doesn't seem too picky, but I would only use the imperative type sentences (e.g. "download the signature public keys..." in the procedures. Here's just a suggestion. Of course, it's up to you to decide if this works. 

```suggestion
After you sign and approve collections, you must download the signature public keys from the {HubName} UI before you add it to the local system keyring.
```

Comment (lmalivert) [reply downstream/modules/automation-hub/proc-downloading-signature-public-keys.adoc]: Thank you, @jself-sudoku! Please review and squash and merge if you are satisfied w/the changes.

File: downstream/modules/automation-hub/proc-downloading-signature-public-keys.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+After you sign and approve collections, you must download the signature public keys from the {HubName} UI before you add it to the local system keyring.
+
+.Procedure
+
+. Log in to your private {HubName} instance in the {HubName} UI.
+. In the left navigation, click *Signature Keys*. The Signature Keys dashboard displays a list of multiple keys: collections and container images.
+
+. Select the menu icon, click btn:[Download Key] to download the public key.
++
```
Comment (jself-sudoku) [inline downstream/modules/automation-hub/proc-downloading-signature-public-keys.adoc:15]: Should this verification step be its own step and not a sub-step to downloading the public key?

File: downstream/modules/automation-hub/proc-downloading-signature-public-keys.adoc [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+= Downloading signature public keys
+
+After you sign and approve collections, you must download the signature public keys from the {HubName} UI before you add it to the local system keyring.
+
+.Procedure
+
+. Log in to your private {HubName} instance in the {HubName} UI.
+. In the left navigation, click *Signature Keys*. The Signature Keys dashboard displays a list of multiple keys: collections and container images.
+
+. Select the menu icon, click btn:[Download Key] to download the public key.
```
Comment (jself-sudoku) [inline downstream/modules/automation-hub/proc-downloading-signature-public-keys.adoc:14]: These seem more like two steps to me (suggestion only):

1. Select the three dots or Kebab menu icon to the right of your public key.
2. Click btn:[Download Key] to download the public key.

Or you could combine it into one sentence: Select the Kebab menu and click btn:[Download Key] to download the public key.

However, I have another suggestion to combine things by having the customer select one of the 3 ways they can download a public key. See below.

File: downstream/modules/automation-hub/proc-downloading-signature-public-keys.adoc [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
++
+* To verify collections, download the key prefixed with `collections-`.
+* To verify container images, download the key prefixed with `container-`.
+
+[NOTE]
+====
+There are two additional ways to download the public key:
+
+* Click the *Copy to clipboard* icon.
+* Click the drop-down menu under the *Publick key* tab and copy the entire public key block.
```
Comment (jself-sudoku) [inline downstream/modules/automation-hub/proc-downloading-signature-public-keys.adoc:24]: Just a minor typo.

```suggestion
* Click the drop-down menu under the *Public key* tab and copy the entire public key block.
```

File: downstream/modules/automation-hub/proc-downloading-signature-public-keys.adoc [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+. Log in to your private {HubName} instance in the {HubName} UI.
+. In the left navigation, click *Signature Keys*. The Signature Keys dashboard displays a list of multiple keys: collections and container images.
+
+. Select the menu icon, click btn:[Download Key] to download the public key.
++
+* To verify collections, download the key prefixed with `collections-`.
+* To verify container images, download the key prefixed with `container-`.
+
+[NOTE]
```
Comment (jself-sudoku) [inline downstream/modules/automation-hub/proc-downloading-signature-public-keys.adoc:19]: Just a suggestion. These two additional methods of downloading a public key seem too important to have as a note. Is it possible to change the step that begins with "Select the menu icon, click **Download Key**..." to say:

Select one of the following methods to download your public key: 

- Select the menu icon and click btn:[Download Key] to download the public key.
- Select the public key from the list and click the *Copy to clipboard* icon.
- Click the drop-down menu under the *Public key* tab and copy the entire public key block.

File: downstream/modules/automation-hub/proc-downloading-signature-public-keys.adoc (hunk 3 above)
Comment (lmalivert) [reply downstream/modules/automation-hub/proc-downloading-signature-public-keys.adoc:14]: Thanks, @jself-sudoku for the suggestions.

**Comment by lmalivert** [comment]
@jself-sudoku I've revised the instructions to implement your suggested edits. Please review.

**Comment by jself-sudoku** [comment]
LGTM

**Pull request merged**